src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -1266,7 +1277,7 @@ def __init__(self,
 
         self.resource_group_name = resource_group_name
         self.resource_provider_namespace = resource_provider_namespace
-        self.parent_resource_path = parent_resource_path
+        self.parent_resource_path = parent_resource_path if parent_resource_path else ''
```

Comment:
if you like to do this, then you can remove all occurrences of `self.parent_resource_path or ''` in the same class
Or you back the change out and apply ` self.parent_resource_path or ''` inside `invoke_action`. Either way, let us ensure the code is consistent.